201602_002.md

```diff
@@ -0,0 +1,141 @@
+# 201602 002
+
+A couple of days after the first logged event, Fernando, Miguel and Pedro are in a Ruby meetup. This time there were live demos.
+
+## 1
+
+Miguel's talk was going pretty well. Then he said "Ok, let's do a live demo". I'm taking over the log.
+
+## 2
+
+Apparently the live demo is only Miguel ~~bragging about~~ showing the product website they built.
+
+## 3
+
+30s into the live demo, Pedro's VPN just blocked the access to the showcased product.
+
+## 4
+
+Miguel refuses to do the captcha. He's now trying to use the staging server. Turns out he still needs a captcha.
+
+## 5
+
+Miguel just tried to convince people to use the product so he doesn't have to do the captcha. It was a nice attempt but we want to watch ship sink, Miguel.
+
+## 6
+
+Miguel just accepted his fate. Miguel is going to write the captcha.
+
+## 7
+
+Turns out the captcha involves recognizing water and grass in random pictures, instead of typing.
+Aaaaaand he failed miserabily.
+
+## 8
+
+Pedro is having none of that and decides to join the fun. Pedro is doing the captcha!
+
+## 9
+
+Aaaand Pedro failed.
+
+## 10
+
+Miguel wanted to choose Gold as currency. Miguel just gifted someone Silver.
+
+## 11
+
+Miguel just found a bug. It's Pedro's fault. Now it's Miguel's fault. Now it's Pedro's fault. Now it's the designer's fault.
+
+## 12
+
+It's Pedro's turn up at bat.
+
+Pedro does not want to offend the public, but offends the public by saying he wanted a more experienced public.
+
+## 13
+
+Pedro blames the computer for spontaneously advancing his slides. Pedro doesn't know how computers work.
```

Comment:
The second sentence should be something like `Pedro doesn't know how to computer`, following our initial style. And it should be on a different line.